src/App.svelte

```diff
@@ -1,372 +1,26 @@
-	const onKeyPress = e => {
-		if (e.charCode === 13) addTask(inputField.value);
-	}
-
-	// function to clear all tasks
-	function clearAllTasks() {
-		if (tasks.length != 0) {
-			tasks = [];
-			refreshAllTasks();
-			if (alertsOn) {
-				alert("All done! Time for a drink! 🍸")
-			}
-		}
-	}
-
-	// function to mark a task as complete, or return it to the array
-	function completeTask(taskToComplete) {
-		let compliments = ["Nice going!", 
-						   "Nice job!", 
-						   "Look at you go!",
-						   "You did it!", 
-						   "Take a bow!", 
-						   "Wow!", 
-						   "Woohoo!", 
-						   "Time for a Mai Tai! 🍹"];
-		let index = Math.floor(Math.random() * compliments.length);
-
-		if (!taskToComplete.done) {
-			if (alertsOn) {
-				alert(compliments[index]);
-			}
-
-			taskToComplete.done = true;
-		} else {
-			taskToComplete.done = false;
-		}
-
-		// need to do this to refresh the styling
-		tasks = [...tasks];
-		refreshAllTasks();
-	}
-
-	// function to remove the specified task from the array
-	function deleteTask(taskToDelete) {
-		tasks = tasks.filter(row => row != taskToDelete);
-		refreshAllTasks();
-	}
-
-	// function to filter active tasks
-	function filterActiveTasks() {
-		mode = "active";
-	}
-
-	// function to filter all tasks
-	function filterAllTasks() {
-		mode = "all";
-	}
-
-	// function to filter active tasks
-	function filterCompletedTasks() {
-		mode = "completed";
-	}
-
-	// function to refresh the active tasks array
-	function refreshAllTasks() {
-		activeTasks = tasks.filter(r => !r.done);
-		completedTasks = tasks.filter(r => r.done);
-	}
-
-	// function to select a random title
-	function selectRandomTitle() {
-		let titles = ["To-Do List",
-					  "Aufgabenliste",
-					  "Lista de Pendientes",
-					  "Liste de Choses à Faire",
-					  "Lista di Cose da Fare",
-					  "Lista de Afazeres",
-					  "Список Дел",
-					  "待办事项",
-					  "タスク一覧",
-					  "רשימת המטלות",
-					  "🍈🍈🍈" //get it, honeydew? honey do? i'm hilarious...
-					  ];
-		let index = Math.floor(Math.random() * titles.length);
-		return titles[index];
-	}
-
-	$: title = selectRandomTitle();
-
-	// function to toggle alerts
-	function toggleAlerts() {
-		alertsOn = !alertsOn;
-	}
+  // flag to handle alert status
+  let alertsOn = true;
 </script>
 
-<body>
-	<main>
-		<h1>{title}</h1>
-		<h2>
-			<table class="centered">
-				<td>
-					<ToolTip title="Type something that needs to get done, then press Enter or click ✔️ below to add it to the list.">
-						<input name="inputField1" class="italic" type="text" bind:this={inputField} bind:value={input} placeholder="What needs to be done?" on:keypress={onKeyPress} size="25">
-					</ToolTip>
-				</td>
-				<td>
-					<ToolTip title="Click to add this new task to the list.">
-						<button on:click={addTask(inputField.value)}><Emoji symbol="✔️" label="Add Task"></Emoji></button>
-					</ToolTip>
-				</td>
-			</table>
```

Comment:
Nowadays, we tend to stay away from `<table>` elements. They're a little wonky to style and layout. Also, you can accomplish most of what you're trying to do with CSS [Grids](https://developer.mozilla.org/en-US/docs/Learn/CSS/CSS_layout/Grids) or [Flexbox](https://developer.mozilla.org/en-US/docs/Learn/CSS/CSS_layout/Flexbox).